js/web/lib/wasm/jsep/webgpu/ops/3rd-party/conv_backprop_mm_webgpu.ts

```diff
@@ -130,7 +130,7 @@ const conv2dTransposeCommonSnippet =
       return ${type}(0.0);
       `;
 
-      const applyActivation = getActivationSnippet(attributes, type);
+      const applyActivation = getActivationSnippet(attributes, type, 'f32');
```

Comment:
Currently, this op (`Conv2DTransposeMatMul`) is only `f32`. See [here](https://github.com/microsoft/onnxruntime/blob/08fbd11825889773fe8d5d3c91bdd473cade7f3d/js/web/lib/wasm/jsep/webgpu/ops/3rd-party/conv_backprop_mm_webgpu.ts#L247-L249) and [here](https://github.com/microsoft/onnxruntime/blob/08fbd11825889773fe8d5d3c91bdd473cade7f3d/js/web/lib/wasm/jsep/webgpu/ops/3rd-party/conv_backprop_mm_webgpu.ts#L36). So I directly pass `f32`. It can be refactored when we enable f16 for this op.

Reply:
Can we make 'f32' default baseType input to getActivationSnippet to avoid hardcoding here and possible other places?

Reply:
Done, thanks.